src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int top;
+    private final int base;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int top, int base, int height) {
+        super(color);
+        this.top = top;
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (Math.round((float) ((top + base) * height * 10) / 2)) / 10;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Additionally, avoid using magic numbers and round-off logic within your calculations. Define a constant for the multiplication factor if necessary.